src/ape_ethereum/ecosystem.py

```diff
@@ -31,6 +31,7 @@
 from ape.contracts._utils import LogInputABICollection
 from ape.exceptions import DecodingError, OutOfGasError, SignatureError, TransactionError
 from ape.types import AddressType, ContractLog
+from ape.utils import to_address
```

Comment:
Curious if we should just move that import here (and no need to rename it anymore)

Then later in v0.2.0 we would delete it from utils, maybe wrap a "deprecation warning" around it

Reply:
There are other breaking changes in this PR (new abstract methods are being added). Could we delete it now then?

Reply:
> There are other breaking changes in this PR (new abstract methods are being added). Could we delete it now then?

Yes!